src/components/UserDetails/UserTags.tsx

```
@@ -0,0 +1,214 @@
+          <tbody className={styles.peopleTabUserTagTableBody}>
+            {displayTags.map((tag) => (
+              <tr key={tag.id} className={styles.peopleTabUserTagTableRow}>
+                <td className={styles.peopleTabUserTagTableCell}>{tag.name}</td>
+                <td className={styles.peopleTabUserTagTableCell}>
+                  {tag.assignedTo}
+                </td>
+                <td className={styles.peopleTabUserTagTableCell}>
+                  {tag.createdOn}
+                </td>
+                <td className={styles.peopleTabUserTagTableCell}>
+                  <a className={styles.peopleTabUserTagCreatedByButton}>
```

Biome (2.1.2)

[error] Provide a href attribute for the a element.

An anchor element should always have a href
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)